specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-07-02-preview/managedClusters.json

```diff
@@ -4432,7 +4432,8 @@
           "type": "string",
           "enum": [
             "Enabled",
-            "Disabled"
+            "Disabled",
+            "SecuredByPerimeter"
```

Comment:
Please document this in x-ms-enum below like I mentioned in the design review:
```
      "x-ms-enum": {
        "name": "PublicNetworkAccess",
        "modelAsString": true,
        "values": [
          {
            "value": "Enabled",
            "description": "<docs>"
          },
          {
            "value": "Disabled",
            "description": "<docs>"
          },
          {
            "value": "SecuredByPerimeter",
            "description": "<docs>"
          }
        ]
      },
```

Reply:
fixed.